lua/grpc.lua

```diff
@@ -24,6 +24,9 @@ end
 if GRPC.port == nil then
   GRPC.port = 50051
 end
+if GRPC.debug == true then
+  GRPC.debug = true
+end
```

Comment:
Isn't this redundant? If a == true then a = true

Reply:
It is indeed not what I intended. I've update it to:

```lua
GRPC.debug = GRPC.debug == true
```

Which still looks redundant, the idea being that I have to make sure that the data type is a boolean to prevent a crash when passing it into the DLL.